source/Xunit.Gherkin.Quick/Feature.cs

```diff
@@ -49,7 +62,16 @@ internal void Scenario(string scenarioName)
                         .ToArray();
                 }
 
-                matchingStepMethod.method.Invoke(this, methodParamValues);
+                try
+                {
+                    matchingStepMethod.method.Invoke(this, methodParamValues);
+                    Output?.WriteLine($"{parsedStep.Keyword} {parsedStep.Text}: PASSED");
+                }
+                catch
+                {
+                    Output?.WriteLine($"{parsedStep.Keyword} {parsedStep.Text}: FAILED");
+                    throw;
```

Comment:
Ah, never mind my last comment. You are retaining the behavior to stop executing steps since you are throwing here. Didn't see this before and hence that comment. Good on this one.